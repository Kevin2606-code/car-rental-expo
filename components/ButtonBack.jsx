import { View, TouchableOpacity, StyleSheet } from "react-native";
import React from "react";
import { Ionicons } from "@expo/vector-icons";

export default function ButtonBack({ onPress }) {
  return (
    <View style={styles.container}>
      <TouchableOpacity onPress={onPress}>
        <Ionicons name="arrow-back-outline" size={24} color="black" />
      </TouchableOpacity>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flexDirection: "row",
    paddingLeft: 20,
  },
});
