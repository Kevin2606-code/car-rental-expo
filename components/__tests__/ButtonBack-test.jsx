import * as React from "react";
import renderer from "react-test-renderer";

import ButtonBack from "../ButtonBack";

it(`ButtonBack renders correctly`, () => {
  const tree = renderer.create(<ButtonBack />).toJSON();

  expect(tree).toMatchSnapshot();
});
