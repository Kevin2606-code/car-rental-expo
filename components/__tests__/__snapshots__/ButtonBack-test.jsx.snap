// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonBack renders correctly 1`] = `
<View
  style={
    {
      "flexDirection": "row",
      "paddingLeft": 20,
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
</View>
`;

exports[`CarList renders correctly 1`] = `
<Text
  style={
    {
      "color": "#ffffff",
      "fontFamily": "PoppinsBold",
      "fontSize": 14,
    }
  }
>
  Location Loading.....
</Text>
`;
